src/functions-and-arrays.js

```diff
@@ -1,13 +1,39 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a
+  };
+  if (a < b) {
+    return b
+  };
```

Comment:
You could also do it as an if-else, or just an if. The equality can be handled as one of the two cases, in the example below, if they are equal, b will be returned. The function ends when the return keyword is reached. What comes after return is not executed so this is why we can do without an else.

```
function maxOfTwoNumbers(a, b) {
  if (a > b) {
    return a;
  }
  return b;
 }
```